_includes/head.html

```diff
@@ -9,11 +9,11 @@
 
   <link rel="canonical" href="{{ site.url }}" />
 
-  <link rel="shortcut icon" type="image/png" href="img/favicon-32x32.png"/>
+  <link rel="shortcut icon" type="image/png" href="{{ site.baseurl }}/img/favicon-32x32.png"/>
 
   <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
   <link rel="stylesheet" type="text/css" href="{{ site.baseurl }}/css/style.css" >
-  <link rel="stylesheet" type="text/css" href="/css/font-awesome.min.css">
+  <link rel="stylesheet" type="text/css" href="{{ site.baseurl }}/css/font-awesome.min.css">
```

Comment:
We should make it consist here either we use `{{ site.basurl }}` or forward stash to be relative to root. We should avoid mixing both and ends in confusion.